repos/system_upgrade/common/libraries/config/version.py

```diff
@@ -18,12 +18,16 @@
 }
 
 
+def get_major_version(version):
+    return version.split('.')[0]
+
+
 def get_source_major_version():
-    return api.current_actor().configuration.version.source.split('.')[0]
+    return get_major_version(api.current_actor().configuration.version.source)
 
 
 def get_target_major_version():
-    return api.current_actor().configuration.version.target.split('.')[0]
+    return get_major_version(api.current_actor().configuration.version.target)
```

Comment:
Just a nitpick. I've just realized we have forgotten to add the docstrings to the functions. Would you mind to add them? So people will understand for sure what is the major version. e.g.

```python
def get_major_version(version):
    """
    Return the major version from the string given string.

    Versioning schema: MAJOR.MINOR.PATCH
    It doesn't matter how many dots are present. Everything until the first dot is return. E.g.:
       8.1.0 => 8
       7.9    => 7
       7       => 7
    """
    return version.split('.')[0]


def get_source_major_version():
    """
    Return the major version of the source (original) system
    """
    return get_major_version(api.current_actor().configuration.version.source)


def get_target_major_version():
    """
    Return the major version of the target system
    """
    return get_major_version(api.current_actor().configuration.version.target)
```

Reply:
It seems ok otherwise.

Reply:
Thanks :) Approved.